### PR TITLE
verticalflip

### LINKED ISSUE #21337: vflip

### PR DESCRIPTION
**Pull request by ManoBharathi93**
Close #21337

### CODE REVIEW

File: ivy/functional/frontends/paddle/vision/transforms.py [hunk 1]
```diff
@@ -13,3 +13,8 @@
 def to_tensor(pic, data_format="CHW"):
     array = ivy.array(pic)
     return Tensor(array)
+
+@to_ivy_arrays_and_back
+def vflip(img):
+  x= ivy.array(img)
+  return ivy.flip(x,axis=0)
```
Comment (a0m0rajab) [inline ivy/functional/frontends/paddle/vision/transforms.py:20]: Thanks for the PR, how do you feel about this suggestion? 

```suggestion
  return ivy.flip(ivy.array(img),axis=0)
```

Note: I am not an ivy member.

Comment (ManoBharathi93) [reply ivy/functional/frontends/paddle/vision/transforms.py:20]: it's great and minimum line. I am happy to commit your suggestion.

Comment (a0m0rajab) [reply ivy/functional/frontends/paddle/vision/transforms.py]: You are welcome.

**Comment by a0m0rajab** [comment]
For the linting test, you can fix that if you went through [this documentation.](https://unify.ai/docs/ivy/overview/deep_dive/formatting.html#pre-commit)

**Comment by ManoBharathi93** [comment]
Why many of them committing to my branch !

**Comment by a0m0rajab** [comment]
I think it's because you did pull the main this to your PR, you can see that they did not commit to your branch but it was commits pulled from the main branch. 

image

**Comment by a0m0rajab** [comment]
It gives a 14 changed files as well, can you check it and fix it? 

image

**Comment by ManoBharathi93** [comment]
> It gives a 14 changed files as well, can you check it and fix it?
> 
> image

can i close this pull request and give another pr

**Comment by a0m0rajab** [comment]
If that's make you feel comfortable then sure! If you like to get familiar with github and tackle this a bit you can try that.
it's up to you.

**Comment by ManoBharathi93** [comment]
I can revert this.. but to save my time let me close and open another pr..

**Pull request closed without merging**